src/components/LocationSearch.stories.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import LocationSearch from "./LocationSearch";
+const toLocation = {};
+export default {
+  title: "Location Search",
+  component: LocationSearch,
+};
+export const Primary = () => (
+  <LocationSearch
+    type="To"
```

Comment:
Is type used?

Reply:
<img width="681" alt="Screenshot 2022-12-19 at 5 31 00 PM" src="https://user-images.githubusercontent.com/113214385/208421587-6247b86b-bc46-455f-a384-93f96786e286.png">

Reply:
What is the difference between label and placeholder. Should we use one of placeholder and type. ALso lets avoid inline styles like marginLeft, instead use marginLeft property.

Reply:
Updated